src/components/login/Login.vue

```diff
@@ -209,6 +216,11 @@
             this.rememberUser = checked;
             localStorage.setItem('rememberUser', checked);
         },
+    },
+    watch: {
+        customError(val) {
+            this.error = val;
+        }
```

Comment:
@osamaamer2024 Making this reactive would mean that even if there's no **real** error and `customError` changed, it will show an error. Is there a reason or a use case where we need `customError` to be reactive?

Reply:
This is not an optimal solution as the prop is already reactive, and a watcher is expensive. I think the issue you're facing could be fixed with `this.$nextTick` which will wait for the dom to update and will display the correct value without a watcher. So please update it accordingly, and apply it as well to the other login methods.